homeassistant/components/geocaching/coordinator.py

```diff
@@ -48,8 +52,14 @@ async def async_token_refresh() -> str:
             update_interval=UPDATE_INTERVAL,
         )
 
-    async def _async_update_data(self) -> GeocachingStatus:
+    async def fetch_new_status(self) -> GeocachingStatus:
+        """Fetch the latest Geocaching status."""
         try:
             return await self.geocaching.update()
+        except GeocachingInvalidSettingsError as error:
+            raise UpdateFailed(error) from error
```

Comment:
We should not send over the bare error message. We prefer to provide a user-facing message (that is translatable), the underlying message could be passed in that case additionally for context.